### PR TITLE
BaSP-TG-21: Modal share component (fixed branch)

### PR DESCRIPTION
**Pull request by MatiasPazConti**
Because of the previous Modal-share-component branch having commits that goes in other branches, this new branch is made to only have Modal commits and not confuse the QA team in future reviews

### CODE REVIEW

File: src/Components/Share/Table/index.js [hunk 1]
```diff
@@ -67,11 +74,49 @@ const Table = ({ headers, data }) => {
                   <td className={styles.buttonsContainer}>
                     <button
                       className={styles.editBtn}
-                      onClick={() => history.push(`/${entitie}/form/${row._id}`)}
+                      onClick={() => {
+                        /*  Verdadera función del botón */
```
Comment (kolodani) [inline src/Components/Share/Table/index.js:78]: you can delete this comments (and are in spanish too)

File: src/Components/Share/Modal/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const Modal = ({ children, showModal, closeModal }) => {
+  if (!showModal) {
+    return null;
+  }
+
+  return (
+    <div className={styles.modalOverlay}>
+      <div className={styles.modalWrapper}>
+        <button className={styles.closeButton} onClick={() => closeModal()}>
```
Comment (kolodani) [inline src/Components/Share/Modal/index.js:12]: we decided to make and empty modal, all the information will be send in the children. remove this button x

Comment (MatiasPazConti) [reply src/Components/Share/Modal/index.js:12]: The button will be removed when all the messages are made

Comment (MatiasPazConti) [reply src/Components/Share/Modal/index.js:12]: Deleted

File: src/Components/Share/Table/index.js (hunk 1 above)
Comment (MatiasPazConti) [reply src/Components/Share/Table/index.js:78]: Deleted!

**Pull request merged**